### ISSUE TITLE
Change default branch to `main`

### ISSUE THREAD

**Issue by trallard** [Quansight/.github#14]
I suggest we move from master to main - this is pretty straightforward, but we might need to do some link updating in downstream repos

**Comment by rgommers** [comment]
Makes sense to me. @asmeurer did this for all repos via a script a while back, maybe this one was just missed because the name starts with a period?

**Comment by asmeurer** [comment]
I added the code of conduct to repos. I didn't do anything with moving master to main. If you change the branch on this repo you'll need to update every code of conduct link in Quansight and Quansight-Labs. You might be able to modify [my script](https://github.com/Quansight/.github/blob/master/add_code_of_conduct_to_repos.py) (by the way, I never was able to get the script running for Quansight repos because of permissions issues, so you maybe should do that too)

**Comment by rgommers** [comment]
Argh, confused the two things. That would be a real pain to update - too many repos.
